### PR TITLE
Monitoring support for the artipie

### PR DESCRIPTION
**Pull request by ChGen**
Monitoring support for the artipie. Part of https://github.com/artipie/benchmarks/issues/60
Implementation consist of two docker-compose parts in `benchmarks/loadtests/monitoring/`:
- `server` - server part, currently deployed on `http://central.artipie.com:3000`. Contains Grafana+InfluxDB2+Telegraf.
- `client` - client part currently deployed on dedicated PC test bench under NAT network. Contains Artipie instance + Prometheus. `Prometheus` regularly collects `artipie` metrics and pushes to `server` to the `telegraf` public port.
Also documentation and scripts were updated.

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review.

File: loadtests/monitoring/server/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+version: "3.7"
+services:
+  telegraf:
+    image: telegraf:1.24.3
+    volumes:
+      - ./conf/telegraf.conf:/etc/telegraf/telegraf.conf:ro
+    depends_on:
+      - influxdb2
```
Comment (dgarus) [inline loadtests/monitoring/server/docker-compose.yml:9]: How to get influxdb2 port here?

File: loadtests/README.md [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
 ## JMeter notes
 * Note that JMeter in Ubuntu 22.04 LTS Linux repository is very old and some modules are missing
 * Note that JMeter may go crazy when recording requests from localhost to localhost. One may need to use external IP.
 * JMeter HTTP proxy mode recording could be done with GUI only. 
 * At the moment we use the following version of JMeter:
 `https://dlcdn.apache.org/jmeter/binaries/apache-jmeter-5.5.tgz`
 
+## JFR and debugging support
+
+JFR recording for running artipie docker container could be accomplised via remote JMX connections. Both, `artipie-docker.sh` and `monitoring/client` open port `9999` for JMX connections to artipie container. Java debugger could be attached via 8000 port.
```
Comment (olenagerasimova) [inline loadtests/README.md]: accomplised  -> accomplished

File: loadtests/monitoring/client/artipie/var/repo/_api_permissions.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+endpoints:
```
Comment (olenagerasimova) [inline loadtests/monitoring/client/artipie/var/repo/_api_permissions.yaml]: this file is not needed

File: loadtests/monitoring/client/artipie/var/repo/_permissions.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+permissions:
+  artipie:
```
Comment (olenagerasimova) [inline loadtests/monitoring/client/artipie/var/repo/_permissions.yaml]: this file is not needed

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@ChGen thanks, just several comments

**Comment by ChGen** [comment]
@dgarus , please review, I moved InfluxDB2 port number to the variable and fixed few other issues.

**Review by dgarus — APPROVED** [review]

**Review by Swizbiz — APPROVED** [review]
@ChGen Thank you! 
LGTM

**Comment by ChGen** [comment]
@olenagerasimova , please review. I fixed the issues, and GitHub requests your approval.

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**